src/main/app/src/app/shared/material-form-builder/form-components/medewerker-groep/medewerker-groep.component.html

```diff
@@ -1,41 +1,97 @@
 <!--
-  ~ SPDX-FileCopyrightText: 2022 Atos
+  ~ SPDX-FileCopyrightText: 2022 Atos, 2024 Lifely
   ~ SPDX-License-Identifier: EUPL-1.2+
   -->
 
 <fieldset>
-  <legend>{{data.label | translate}}</legend>
+  <legend>{{ data.label | translate }}</legend>
   <mat-form-field appearance="fill" class="full-width" floatLabel="always">
-    <mat-label>{{data.groepLabel| translate}}</mat-label>
+    <mat-label>{{ data.groepLabel | translate }}</mat-label>
     <input
-      placeholder="{{(data.required ? 'groep.-kies-' : 'groep.-geen-') | translate}}"
+      placeholder="{{
+        (data.required ? 'groep.-kies-' : 'groep.-geen-') | translate
+      }}"
       [formControl]="data.groep"
       [id]="data.id + '_groep'"
       [matAutocomplete]="groepAC"
       matInput
       type="text"
       [maxlength]="data.maxlength"
     />
+    <mat-icon
+      matSuffix
+      *ngIf="!hasGroep"
+      [ngClass]="{
+        error:
+          data.groep.invalid &&
+          (data.formControl.touched || data.formControl.dirty)
+      }"
+      >search</mat-icon
+    >
+    <button
+      *ngIf="hasGroep"
+      type="button"
+      mat-icon-button
+      matSuffix
+      aria-label="Maak veld leeg"
+      (click)="clearField(data.groep)"
+    >
+      <mat-icon
+        [ngClass]="{
+          error:
+            data.groep.invalid &&
+            (data.formControl.touched || data.formControl.dirty)
+        }"
+        >clear</mat-icon
+      >
+    </button>
     <mat-autocomplete #groepAC="matAutocomplete" [displayWith]="displayFn">
       <mat-option *ngFor="let groep of filteredGroepen | async" [value]="groep"
         >{{ groep.naam }}</mat-option
       >
     </mat-autocomplete>
     <mat-error appEnhanceMatError *ngIf="data.groep.invalid"
-      >{{getMessage(data.groep, 'Groep')}}</mat-error
+      >{{ getMessage(data.groep, "Groep") }}</mat-error
     >
   </mat-form-field>
   <mat-form-field appearance="fill" class="full-width" floatLabel="always">
-    <mat-label>{{data.medewerkerLabel | translate}}</mat-label>
+    <mat-label>{{ data.medewerkerLabel | translate }}</mat-label>
     <input
-      placeholder="{{'behandelaar.-geen-' | translate}}"
+      placeholder="{{ 'behandelaar.-geen-' | translate }}"
       [formControl]="data.medewerker"
       [matAutocomplete]="medewerkerAC"
       [id]="data.id + '_medewerker'"
       matInput
       type="text"
       [maxlength]="data.maxlength"
     />
+    <mat-icon
+      matSuffix
+      *ngIf="!hasMedewerker"
+      [ngClass]="{
+        error:
+          data.medewerker.invalid &&
+          (data.formControl.touched || data.formControl.dirty)
+      }"
+      >search</mat-icon
+    >
+    <button
+      *ngIf="hasMedewerker"
+      type="button"
+      mat-icon-button
+      matSuffix
+      aria-label="Maak veld leeg"
```

Comment:
Translation instead of static value please

Reply:
True; on it

Reply:
Fixed ✅